app/services/link_agency_identities.rb

```diff
@@ -0,0 +1,42 @@
+class LinkAgencyIdentities
```

Comment:
It'd be nice to see this output the table of new UUIDs that we would end up having to provide to the agencies.

Reply:
I was planning to provide an adhoc db query.  agency | old uuid | new uuid.  for a few of the agencies they would not have this mapping depending on priority.